lua/otter/tools/extensions.lua

```diff
@@ -10,6 +10,4 @@ return {
   javascript = "js",
   yaml = "yml",
   sql = "sql",
-  elixir = "ex",
-  markdown = "md"
```

Comment:
You reverted https://github.com/jmbuhr/otter.nvim/pull/64 here

Reply:
oh, sorry! Got lost in all the formatting I also hadn't merged into the branch yet.